magic_script.py

```diff
@@ -0,0 +1,265 @@
+import json
+import datetime
+#import pytz
+import os
+#from pytz import timezone
+from datetime import datetime
+import logging
+import boto3
+from sys import argv
+from botocore.exceptions import ClientError
+import sys
+import dropbox
+from boxsdk import OAuth2, Client
+
+
+CONFIGFILE = "/mnt/gsoc/test/s3/config.json"

```

Comment:
Please do not hardcode paths like this in your scripts.  This should assume a relative path of the present directory.